components/authn-service/tokens/integration/token_integration_test.go

```diff
@@ -61,14 +85,9 @@ func TestChefClientAuthn(t *testing.T) {
 		t.Fatalf("parse test upstream URL %+v: %v", upstream, err)
 	}
 
-	pgURLGiven := false
-	pgCfg := pg.Config{
-		PGURL:          constants.TestPgURL,
```

Comment:
is this what was passing the `postgres` user? 🤔

Reply:
Yep